matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -269,6 +272,19 @@ private static void createBranches(MatpowerModel model, ContainersMapping contai
                         .setG(0)
                         .setB(mBranch.getB() / zb)
                         .add();
+                if (mBranch.getPhaseShiftAngle() != 0) {
```

Comment:
According to @zamarrenolm, the convention of angle increment sign has changed at version 3.2 of matpower files. After the version 3.2, the convention is the same as PSSE file:
`
    private static void tapChangerToPhaseTapChanger(TapChanger tapChanger, PhaseTapChangerAdder ptc) {
        ptc.setLowTapPosition(0)
            .setTapPosition(tapChanger.getTapPosition());

        tapChanger.getSteps().forEach(step ->
            ptc.beginStep()
                .setRho(1 / step.getRatio())
                .setAlpha(-step.getAngle())
                .setR(step.getR())
                .setX(step.getX())
                .setB(step.getB1())
                .setG(step.getG1())
                .endStep());
        ptc.add();
    }
`
And before, it was just the opposite :-)

Reply:
I generated a small case in `matpower .M` format (ascii format) starting from the `IEEE14` case and by changing a transformer to a phase shifter.
I solved the case on matpower and I exported the results as a `matpower .mat` format file.
Then I imported this file on this branch and I exported to `xiidm` format.
I ran the obtained `xiidm` case on `HELM` and on `open-Loadflow` obtaining the same result but different from the matpower one.
Then I modify the code to 

```
                    newTwt.newPhaseTapChanger()
                            .setTapPosition(0)
                            .beginStep()
                            .setRho(1)
                            .setAlpha(-mBranch.getPhaseShiftAngle())
                            .setR(0)
                            .setX(0)
                            .setG(0)
                            .setB(0)
                            .endStep()
                            .add();

```
After the modificación all three results are identical.
So the `alpha `attribute should be the opposite of the `mBranch.getPhaseShiftAngle()`.
The change is in concordance with the matpower documentation.

Reply:
Thanks a lot for your analysis, and your test makes us confident for the implementation. I have made the fix and I have added a fictitious test based on IEEE 14 test case. Just tell me if you agree with my modifications.

Reply:
I agree with your modification.
You added a new test that uses the resource `ieee14-phase-shifter.json` but I cannot find this resource. Maybe you have to commit it?
If you add the resource I will also check the test.

Reply:
Done, thank you it was a mistake!

Reply:
The test case works fine. I ran the `ieee14-phase-shifter` case on matpower and I checked that we get the same result on HELM and open-Loadflow. 
I will prepare an additional commit to update the voltage and angle attributes of the  `ieee14-phase-shifter.json` and `ieee14-phase-shiftermat.xiidm` resources with the values I got after solving the case on matpower. Do you agree?

Reply:
> According to @zamarrenolm, the convention of angle increment sign has changed at version 3.2 of matpower files. After the version 3.2, the convention is the same as PSSE file: ` private static void tapChangerToPhaseTapChanger(TapChanger tapChanger, PhaseTapChangerAdder ptc) { ptc.setLowTapPosition(0) .setTapPosition(tapChanger.getTapPosition());
> 
> ```
>     tapChanger.getSteps().forEach(step ->
>         ptc.beginStep()
>             .setRho(1 / step.getRatio())
>             .setAlpha(-step.getAngle())
>             .setR(step.getR())
>             .setX(step.getX())
>             .setB(step.getB1())
>             .setG(step.getG1())
>             .endStep());
>     ptc.add();
> }
> ```
> 
> ` And before, it was just the opposite :-)

According to https://matpower.org/docs/ref/matpower5.0/caseformat.html (and confirmed by our use cases), mat file version could be 1 or 2. @zamarrenolm @annetill are you sure about the version 3.2 as a breaking change version for phase shifter convention?

Reply:
Version 3.2 refers to MATPOWER software, not to file format. As you commented there are two versions of the MATPOWER file format (version 1 and version 2). 
MATPOWER software versions 3.2 and subsequent  use version 2 format. 
MATPOWER software versions 3.0 and earlier use version 1 format. 
Here is the MATPOWER software release history:
```
Pre 1.0, released Jun 25, 1997
Version 1.0, released Sep 17, 1997
Version 1.0.1, released Sep 19, 1997
Version 2.0, released Dec 24, 1997
Version 3.0, released Feb 14, 2005
Version 3.2, released Sep 21, 2007
Version 4.0, released Feb 7, 2011
Version 4.1, released Dec 14, 2011
Version 5.0, released  Dec 17, 2014
Version 5.1, released Mar 20, 2015
Version 6.0, released Dec 16, 2016
Version 7.0, released Jun 20, 2019
Version 7.1, released Oct 8, 2020
```

The sign convention used for phase shifters has been changed in version 3.2 to be consistent with PTI.